api/test/unit/models/media_report_test.py

```diff
@@ -0,0 +1,138 @@
+import uuid
+from typing import Literal, Union
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from catalog.api.models import (
+    Audio,
+    AudioReport,
+    DeletedAudio,
+    DeletedImage,
+    Image,
+    ImageReport,
+    MatureAudio,
+    MatureImage,
+)
+from catalog.api.models.media import DEINDEXED, DMCA, MATURE, MATURE_FILTERED, PENDING
+
+
+pytestmark = pytest.mark.django_db
+
+MediaType = Union[Literal["audio"], Literal["image"]]
+
+
+@pytest.fixture
+def media_obj():
+    def _get_media_obj(type: MediaType):
+        klass = Audio if type == "audio" else Image
```

Comment:
This could potentially be a `dict::get` call, which might make it easier to add future media types:
```suggestion
        klass = {
            "audio": Audio,
            "image": Image,
        }.get(type)
```